ui/src/app/components/connection-profile-form/connection-profile-form.component.ts

```diff
@@ -93,7 +93,7 @@ export class ConnectionProfileFormComponent implements OnInit {
       localStorage.setItem(TargetDetails.Publication, formValue.publication)
     }
     if (this.selectedOption === Profile.NewConnProfile) {
-      let payload: ICreateConnectionProfile = {
+      let payload: ICreateConnectionProfileV2 = {
```

Comment:
I think I'd prefer the `V2` tagging in this case since it disambiguates the evolution of `ICreateConnectionProfile` interface, but I don't have a strong opinion. 
Please chose between `ICreateConnectionProfileV2` and `ICreateConnectionProfileForShard`. @shreyakhajanchi

Reply:
Both co-exist currently, since `ICreateConnectionProfile` is used in the single instance flow and I wanted to be careful not to cause a regression there.

I'll keep this distinction for now, and we can rename it back to `ICreateConnectionProfile` once the single instance and sharded flows are merged together.